packages/css-framework/src/components/_timeline.scss

```diff
@@ -0,0 +1,280 @@
+@use "../abstracts/functions" as f;
```

Comment:
there's a few opportunities to use `f.spacing` in here

Reply:
I'm not sure where else I would use it - could you provide some specific examples.

There is a `16rem` margin used but the spacing helper doesn't scale past `10rem` (`spacing("20")`).